src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java

```diff
@@ -132,6 +132,13 @@ public void cancelTicketTransfer(String token){
         tttRepository.save(ttt);
     }
 
+    @Override
+    public Collection<TicketTransferToken> getTicketTransferTokensByUser(String username) {
```

Comment:
You're not performing a query for all TicketTransferTokens but only the valid ones. Please rename the method to represent the query.